e2e/test/scenarios/dashboard-filters/dashboard-filters-sql-number.cy.spec.js

```diff
@@ -89,7 +87,73 @@ describe("scenarios > dashboard > filters > SQL > text/category", () => {
   });
 });
 
+describe("scenarios > dashboard > filters > SQL > number", () => {
+  const questionDetails = {
+    name: "Question 1",
+    native: {
+      query:
+        "SELECT * from products where true [[ and price > {{price}}]] [[ and rating > {{rating}} ]] limit 5;",
+      "template-tags": {
+        price: {
+          type: "number",
+          name: "price",
+          id: "b22a5ce2-fe1d-44e3-8df4-f8951f7921bc",
+          "display-name": "Price",
+        },
+        rating: {
+          type: "number",
+          name: "rating",
+          id: "b22a5ce4-fe1d-44e3-8df4-f8951f7921bc",
+          "display-name": "Rating",
+        },
+      },
+    },
+  };
+
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createNativeQuestionAndDashboard({ questionDetails }).then(
+      ({ body: { card_id, dashboard_id } }) => {
+        visitQuestion(card_id);
+
+        visitDashboard(dashboard_id);
+      },
+    );
+
+    editDashboard();
+  });
+
+  it("should keep filter value on blur", () => {
```

Comment:
Could you add the issue number in the test name so we know it's a reproduction? e.g.
```suggestion
  it("should keep filter value on blur (metabase#31975)", () => {
```

Reply:
should I move it to repro then?

Reply:
Not necessarily, some repros are inside another feature spec just like this so we don't have to have an overhead of a new Cypress file.

I think in this case, it's reasonable to be in this file. But it should contain the issue number in the test name.